packages/client/src/v2-events/features/events/actions/print-certificate/pdfUtils.ts

```diff
@@ -272,11 +316,52 @@ export function compileSvg({
     if (isEqual($metadata, obj)) {
       return getMixedPath(resolvedMetadata, propertyPath)
     }
-    return getMixedPath(resolvedDeclaration, propertyPath)
+
+    if (isEqual(resolvedDeclaration, obj)) {
+      return getMixedPath(resolvedDeclaration, propertyPath)
+    }
+
+    const action = ActionDocument.safeParse(obj)
+    if (action.success) {
+      const resolvedAction = {
+        id: action.data.id,
+        type: action.data.type,
+        createdAt: DateField.stringify(intl, action.data.createdAt),
+        createdBy: users.find((user) => user.id === action.data.createdBy),
+        createdByUserType: action.data.createdByUserType,
+        createdBySignature: action.data.createdBySignature,
+        createdAtLocation: LocationSearch.stringify(
+          intl,
+          locations,
+          action.data.createdAtLocation
+        ),
+        createdByRole: action.data.createdByRole
+      }
+
+      return getMixedPath(resolvedAction, propertyPath)
+    }
+
+    return obj[propertyPath as keyof typeof obj] ?? ''
```

Comment:
Will`$lookup` work with arrays? If so maybe we should put an example of how the path would look like

Reply:
Not by default, although of course you could do `{{ $lookup ['a'] 0 }}` but I think by default we would only use it for objects